File: C++14/백준/Bronze/6436. Floppies/Floppies.cc
#include <iostream>
#include <cmath>
using namespace std;

int main(void){
    double a;
    int i = 1;
    while(true){
        cin >> a;
        if (a==0){
            break;
        }
        cout << "File #" << i << endl;
        cout << "John needs " << ceil(a/2480000) <<" floppies." << "\n\n";
        i++;
    }
    return 0;
}

File: C#/백준/Bronze/6436. Floppies/Floppies.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int i = 1;
            while(true){
                double a = double.Parse(Console.ReadLine());
                if (a==0){
                    break;
                }
                Console.WriteLine($"File #{i}");
                Console.WriteLine($"John needs {Math.Ceiling(a/2480000)} floppies.\n");
                i++;
            }
        }
    }
}